### DISCUSSION TITLE
Estimate the errors for key parameters

### DISCUSSION THREAD

**Discussion by Codeprey** [mboquien/cigale#59] [Q&A] [answered]
Hello,

I'm using cigale-v2022.1 to fit some SEDs. Now I can get the best fitted parameters after a run. But I also want to estimate the errors for some key parameters (e.g., stellar mass), what should I do? The manual doesn't tell anything about this.

Thanks in advance,

Peter

**Comment by mboquien** [comment, marked as answer]
For this you can use the columns starting with `bayes.`. They give the values of the physical properties. Those with the `_err` suffix give the uncertainties on the corresponding properties.
